app/views/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t("shared.error_message", count_error:
+        pluralize(@user.errors.count, "error"))%>
```

Comment:
https://viblo.asia/p/i18n-in-rails-part-2-GrLZDAO3lk0
Cách thể hiện số ít và số nhiều với i18n
Nghiên cứu em nhé